index.html

```diff
@@ -198,6 +198,29 @@ <h3>Expression Format</h3>
           [=preference=] for the Global Privacy Control value.
         </p>
       </section>
+      <section>
+        <h3>Preference Caching</h3>
+        <p>
+          The [=preference=] MUST be cached on each top-level navigation to ensure consistency in communication of
+          the person's request that their data "not be sold or shared." This means that if the [=preference=] changes
+          during or after a top-level navigation, it WILL NOT be reflected until the next navigation.
```

Comment:
Is "WILL NOT" just emphasis or do the capitals signify something?

```suggestion
          during or after a top-level navigation, it will not be reflected until the next navigation.
```